### PR TITLE
Initial sandbox vars

### PR DESCRIPTION
**Pull request by OmarEdraak**
It's now ready for review:

 - http://edraak.cc

### CODE REVIEW

**Comment by OmarEdraak** [comment]
@shadinaif @devalih fyi

File: playbooks/update_theme.yml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+      when: celery_worker is not defined and not devstack
+      with_items: ['absent', 'directory']
+      tags:
+        - gather_static_assets
+        - assets
+        - update_lms_theme:compile_css
+
+    # Gather assets using paver if possible
+    - name: "gather lms static assets with paver"
+      command: "{{ COMMON_BIN_DIR }}/edxapp-update-assets-lms"
```
Comment (shadinaif) [inline playbooks/update_theme.yml]: @OmarEdraak AFAIK this has changed in Hawthorn to be:
`      command: "{{ COMMON_BIN_DIR }}/edxapp-update-assets"`
which will execute both `lms` and `cms`

File: playbooks/update_theme.yml [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    - name: "gather cms static assets with paver"
+      command: "{{ COMMON_BIN_DIR }}/edxapp-update-assets-cms"
+      when: celery_worker is not defined and not devstack
+      tags:
+        - gather_static_assets
+        - assets
+
+    - name: restart LMS
+      supervisorctl:
+        name: "edxapp:lms"
```
Comment (shadinaif) [inline playbooks/update_theme.yml]: in Hawthorn, this one to be:
`        name: "lms"`

File: playbooks/update_theme.yml [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+        state: restarted
+      become_user: "{{ supervisor_service_user }}"
+      when: celery_worker is not defined and not disable_edx_services
+      tags:
+        - manage
+        - update_lms_theme
+
+    - name: restart CMS
+      supervisorctl:
+        name: "edxapp:cms"
```
Comment (shadinaif) [inline playbooks/update_theme.yml]: in Hawthorn, this one to be:
`        name: "cms"`

File: playbooks/update_theme.yml [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+        config: "{{ supervisor_cfg }}"
+        state: restarted
+      become_user: "{{ supervisor_service_user }}"
+      when: celery_worker is not defined and not disable_edx_services
+      tags:
+        - manage
+
+    - name: ensure edxapp has started
+      supervisorctl:
+        name: "edxapp:"
```
Comment (shadinaif) [inline playbooks/update_theme.yml]: in Hawthorn, this is to be:
`        name: "lms"`
and another task for cms

File: playbooks/update_theme.yml [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+    # Gather assets using paver if possible
+    - name: "gather lms static assets with paver"
+      command: "{{ COMMON_BIN_DIR }}/edxapp-update-assets-lms"
+      when: celery_worker is not defined and not devstack
+      tags:
+        - gather_static_assets
+        - assets
+        - update_lms_theme:compile_css
+
+    - name: "gather cms static assets with paver"
```
Comment (shadinaif) [inline playbooks/update_theme.yml]: so this one is not needed anymore

File: playbooks/update_theme.yml [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+      supervisorctl:
+        name: "edxapp:"
+        supervisorctl_path: "{{ supervisor_ctl }}"
+        config: "{{ supervisor_cfg }}"
+        state: started
+      become_user: "{{ supervisor_service_user }}"
+      when: celery_worker is not defined and not disable_edx_services
+      tags:
+        - manage
+        - update_lms_theme
```
Comment (shadinaif) [inline playbooks/update_theme.yml]: You might need to tweak the tags a bit after changing the code :)

**Comment by shadinaif** [comment]
@OmarEdraak  that solves it too :)

**Comment by OmarEdraak** [comment]
Thanks @shadinaif!

**Pull request merged**